backend/curriculum_tracking/tests/test_activity_log_entry_creators_envoked.py

```diff
@@ -1,14 +1,22 @@
 """for each of the log entry creators in curriculum_tracking.activity_log_entry_creators, make sure it is called when it should be and creates the correct log entries
 """
 from rest_framework.test import APITestCase
-
+from django.test import TestCase
+import curriculum_tracking
 from test_mixins import APITestCaseMixin
 from . import factories
 from core.tests.factories import UserFactory
+from .factories import AgileCardFactory, RecruitProjectFactory, ProjectContentItemFactory
```

Comment:
Why are you importing these when they are already available? You can call `factories.AgileCardFactory` whenever you want